### ISSUE TITLE
Movie Sets do not append 'Collection'

### ISSUE THREAD

**Issue by Reliant73** [croneter/PlexKodiConnect#549]
When Movie Sets are configured in PKC, KODI does not append the word ‘Collection’ to the end of the Movie Set Title e.g. 'Star Wars Collection’ is displaying as just ‘Star Wars’. In Plex it displays correctly as ‘Star Wars Collection’ (See Examples Below). Using KODI 17.6 Krypton.

**KODI Movie Sets with PKC Interation**
![kodi pkc movie sets](https://user-images.githubusercontent.com/44363462/47313258-edef7100-d69a-11e8-956e-362d0328cf8c.jpg)

**KODI Movie Sets With No PKC Integration**
![kodi movie sets no pkc](https://user-images.githubusercontent.com/44363462/47313262-f182f800-d69a-11e8-9c11-50c37fed42be.jpg)

**Link To Orginal Post**
[https://forums.plex.tv/t/plexkodiconnect-kodi-plex-integration-done-right/137555/4022?u=reliant](URL)

**Comment by dazedcrazy** [comment]
Do you use [this?](https://forums.plex.tv/t/metadata-agents-for-exported-xbmc-library/15430) Because i used to have that [problem](https://forums.plex.tv/t/metadata-agents-for-exported-xbmc-library/15430/1498) i just removed the changes made to the agent

**Comment by croneter** [comment]
This is not something I am going to "fix". PKC will provide only the collection name that the PMS uses - as is, without appending or changing anything. 

If you want to "fix" this, you'd need to see that Plex collections are named "correctly", e.g. by manually renaming them. Sorry... 😞

**Comment by Reliant73** [comment]
Hi Croneter,

Thanks for looking at it. Unfortunately It's a bit weird how it works in Plex as I have named them as Collections in Plex e.g: let's take 28 Days/Later Collection.

In the actual collection title I have called it 28 Days/Later Collection

![28 weeks 1](https://user-images.githubusercontent.com/44363462/48183503-c7685000-e393-11e8-8fd5-44aa0fc61fdf.jpg)

However it doesn't look like this is where PKC takes the name from. It appears to take it from the collection tag against the individual Movies.

![28 weeks 2](https://user-images.githubusercontent.com/44363462/48183563-f8e11b80-e393-11e8-85cd-6bcb7aa4ff9e.jpg)

If I update that and append collection that works in in KODI but it then when you look at the movie information in Plex you get something stupid when it now says"Part of the '28 Days/Week Collection Collection"

![28 weeks 3](https://user-images.githubusercontent.com/44363462/48183818-e5828000-e394-11e8-835f-65db0f3dcc23.jpg)


That kinda hurts my eyes so I guess I will have to work out if I want it to say collection in KODI and live with the Plex eyesore or just live without collection appended in KODI :(

Am I doing something wrong that it is doing this?

Cheers

Reliant
